TestResultSummaryService/routes/getPerffarmRunCSV.js

```diff
@@ -2,50 +2,44 @@ const request = require('request');
 const csv = require('csv');
 const fs = require('fs');
 const path = require('path');
-
+const ArgParser = require('../ArgParser');
 module.exports = async ( req, res ) => {
-
-    fs.readFile(path.resolve(__dirname)+'/../perffarm.config', 'utf8', function(err, perfman_credentials){
-
-        let perfman_user = perfman_credentials.split(' ')[0];
-        let perfman_pass = perfman_credentials.split(' ')[1]
-
-        request.get('http://perffarmServer/perfsite/services/parse_results.php?metric=&runids='+req.query.baselineID+'&ignoreExcludes=true&format=csv&groupby=run', {
-            'auth': {
-                'user': perfman_user,
-                'pass': perfman_pass,
-                'sendImmediately': false
-            }}, function(err, res2, baselineBody){
+    const {user, pass} = ArgParser.getConfig() === undefined ? {} : ArgParser.getConfig().perfFarm;
+    request.get('http://perffarm.hursley.ibm.com/perfsite/services/parse_results.php?metric=&runids='+req.query.baselineID+'&ignoreExcludes=true&format=csv&groupby=run', {
```

Comment:
Can this URL be replaced with variable fed to service, for generic approach?  I won't block this PR for it, but will be good to do.

Reply:
and http is.... a little unsecure

Reply:
Changeset is updated. URL is replaced to `perffarmServer`
As discussed, we are going to have a file to capture the internal servers, so that external will not get affected.